File: TEMA2/typescript/aleatorios.ts
import readline from 'node:readline/promises';
import {stdin as input, stdout as output } from 'node:process'
const rl = readline.createInterface({input, output})

function aleatorios(n: number){
    let pares = ""
    let impares = ""
    for (let count=0;count < n;count++) {
        let numero = Math.floor(Math.random() * 100) + 1;
        console.log(numero)
        if (numero % 2 == 0) pares += `${numero}, `;
        else impares += `${numero}, `
    }
    return `Pares: ${pares}\nImpares: ${impares} `
}
    
console.log("********Aleatorios pares impares**********")
const n = await rl.question("Introduzca el valor de n: ")
console.log(aleatorios(n));
rl.close();

File: TEMA2/python/aleatorios.py
import random;
def aleatorios(n):
    count = 0
    pares = ""
    impares = ""
    while count < n: # cambio a for: for count in range(n):
        numero = random.randint(1,100)
        if numero % 2 == 0:
            pares += f"{numero}, "
        else:
            impares += f"{numero}, "
        count +=1
    return f"Pares: {pares}\nImpares: {impares} "

print("********Aleatorios pares impares**********")
print('Introduzca el valor de n: ')
n = int(input())
print(aleatorios(n))